src/ui/ResultLayoutSelector/ResultLayoutSelector.ts

```diff
@@ -75,6 +75,8 @@ export class ResultLayoutSelector extends Component {
      *
      * By default, the mobile mode breakpoint is at 480 px screen width.
      *
+     * To change this default value, use the {@link SearchInterface.options.responsiveSmallBreakpoint} option.
```

Comment:
(...) use the [`responsiveSmallBreakpoint`]{@link SearchInterface.options.responsiveSmallBreakpoint} option.